frontend/integration-tests/views/monitoring.view.ts

```diff
@@ -32,3 +33,66 @@ export const modalConfirmButton = $('#confirm-action');
 export const alertManagerYamlForm = $('.co-alert-manager-yaml__form');
 export const successAlert = $('.pf-m-success');
 export const helpText = $('.co-help-text');
+
+// Configuration Overview
+export const alertRoutingHeader = $$('h2').first();
+export const alertRoutingEditButton = $('.btn-edit-alert-routing');
+
+export const groupByInput = $('#input-group-by');
+export const groupWaitInput = $('#input-group-wait');
+export const groupIntervalInput = $('#input-group-interval');
+export const repeatIntervalInput = $('#input-repeat-interval');
+
+export const groupByValue = $('#group_by_value');
+export const groupWaitValue = $('#group_wait_value');
+export const groupIntervalValue = $('#group_interval_value');
+export const repeatIntervalValue = $('#repeat_interval_value');
+
+export const createReceiverButton = $('#create-receiver');
+export const receiverNameInput = $('#receiver-name');
+
+export const receiverTypeDropdownButton = $('[data-test-id="dropdown-button"]');
+export const pagerDutyReceiverTypeLink = $('#pagerduty_configs-link');
+export const pagerDutyReceiverForm = $('#pagerduty-receiver-form');
+export const routingLabelsEditor = $('#receiver-routing-labels-editor');
+export const pagerdutyKeyLabel = $('#pagerduty-key-label');
+export const pagerdutyPrometheusIntegrationType = $('#integration-type-prometheus');
+export const pagerdutyIntegrationKey = $('#integration-key');
+export const firstRouterLabelName = $('#label-name-0');
+export const firstRouterLabelValue = $('#label-value-0');
+export const disabledKebabMenuItem = $('.pf-c-dropdown__menu-item.pf-m-disabled');
+export const confirmAction = $('#confirm-action');
+export const yamlTextArea = $('textarea');
+
+export const rowForName = (name: string) =>
+  crudView.resourceRows
+    .filter((row) =>
```

Comment:
Avoid `filter` as it can result in `StaleElementReferenceErrors`. Prefer adding a `data-test-id` to the element you need.

Reply:
fixed